examples/vae/main.py

```diff
@@ -18,30 +18,23 @@
 that can be easily tested and imported in Colab.
 """
 
-from absl import app
-from absl import flags
 import jax
 import tensorflow as tf
-
 import train
-
+from absl import app, flags, logging
```

Comment:
google style is usually 1 import per line.  Please run against the "pyink" autoformatter.